tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+minversion=2.6.0
+envlist = py26,py27,py36
+skip_missing_interpreters = True
+
+[testenv]
+usedevelop = true
```

Comment:
Why do we want usedevelop? This would get us the development branches of our pip dependencies. also true ==True?